app_dart/lib/src/service/luci_build_service.dart

```diff
@@ -73,7 +82,6 @@ class LuciBuildService {
     return shards;
   }
 
-  /// Returns an Iterable of try BuildBucket build for a given Github [slug], [sha], [builderName].
```

Comment:
Any reason to remove these docs?

Reply:
Hmm, not sure why this was removed. I will add it back in.